Sugestões da Taty (#22)
R-Ladies-Sao-Paulo/blog: drafts/2023-04-acoes-diversidade/en.qmd

### Space for children and babies

> GuGuDaDados (free translation GuguDadata) is a collaborative and experimental space that RLadies São Paulo proposes with the aim of keeping children entertained and cared for their mothers/fathers/caretakes can participate in community activities.

Author: ```suggestion
GuGuDaDados (free translation GuguDadata) is a collaborative and experimental space that RLadies São Paulo proposes to keep children entertained and cared for, while their mothers/fathers/caretakes can participate in community activities.
```
